### DISCUSSION TITLE
Title is “edit comment” when leaving a comment

### DISCUSSION THREAD

**Discussion by kimroen** [community/community#3937] [Mobile] [answered]
This is what it looks like after having tapped "Comment" on the bottom of a PR:

![image](https://user-images.githubusercontent.com/520420/117843839-76e14e80-b27f-11eb-98b2-4f6c50c4471f.png)

"Edit comment" made me go back and make sure I had tapped the right thing before moving forward with writing the comment I wanted to leave.

Perhaps "Add Comment" or similar would be even better?

**Comment by Iron-Ham** [comment, marked as answer]
Thank you for reporting this. I've opened an issue so that we can track this internally on the team.

**Comment by Aadimeena7372** [comment]
👀 I’m interested in ...
🌱 I’m currently learning ...
💞️ I’m looking to collaborate on ...
📫 How to reach me ...

**Comment by Carl251936** [comment]
:)
